src/pages/index.js

```diff
@@ -460,11 +464,16 @@ Index.defaultProps = {
   data: null,
 }
 
+// we have a top level Catch Boundary because sentry is not handling uncaught exceptions as expected
+// you can see the unsolved issue here: https://github.com/vercel/next.js/issues/1852
+// withSentrySSR sets the user data for our error logger once the user has been authenticated
```

Comment:
Readability request: please wrap comments when they're longer than ~80 characters and use sentences/periods when it's more than just a phrase.